pkgs/by-name/st/starpu/package.nix

```diff
@@ -0,0 +1,107 @@
+{
+  # derivation dependencies
+  lib,
+  fetchzip,
+  stdenv,
+
+  # starpu dependencies
+  hwloc,
+  libuuid,
+  libX11,
+  fftw,
+  fftwFloat, # Same than previous but with float precision
+  pkg-config,
+  libtool,
+  autoconf,
+  automake,
+  simgrid ? null, # Can be null because Simgrid is optional
```

Comment:
I think you could drop `? null` here. If the argument is not used it won't be built or evaluated.